pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -38,6 +42,22 @@ let
     sha256 = "0xw028iqp69j9mxv0kbwraplgkj5i5djdlgf0anpkc5cdbsf96r9";
   };
 
+  aa-teardown = writeShellScript "aa-teardown" ''
```

Comment:
I feel like there ought to be a comment at the start of this script noting that it's NixOS's addition and not an official AppArmor program like the other aa-* tools.  (I might even go so far as to have it print a note to that effect when run, not least to discourage people from reporting problems with it upstream.)

Reply:
I've added a comment explaining that this is like upstream's `aa-teardown`, and why it's much simpler to package it like that.

Reply:
Thanks.  I suggest putting the comment inside the shell script, so that it's visible both to someone reading the Nix script that generates the shell script and to someone reading the generated shell script itself without knowing from where it comes.